src/istio/control/http/attributes_builder.cc

```diff
@@ -67,9 +67,16 @@ void AttributesBuilder::ExtractRequestHeaderAttributes(CheckData *check_data) {
 }
 
 void AttributesBuilder::ExtractAuthAttributes(CheckData *check_data) {
+  utils::AttributesBuilder builder(&request_->attributes);
```

Comment:
Shouldn't you need to update unit test for this, i.e remove this line
https://github.com/istio/proxy/blob/release-1.0/src/istio/control/http/attributes_builder_test.cc#L447

(I don't know why pre-submit tests didn't fail though)

Reply:
Mock stubs were missing. Added and corrected the unit test.